### PR TITLE
Added FAQ page ++

### PR DESCRIPTION
**Pull request by ElleEss**
#### Description
* Added FAQ page
* Added Summary of key features page (linked to from FAQs)
* Put these two docs in a new `/docs/contents/faqs` directory
* Added to animated gifs for the Key Features page
* From /README.md - linked to the FAQ doc quite prominently
* From /docs/README.md - linked to the FAQ doc quite prominently
* From the /docs/README.md - linked to the FAQ doc and the summary of key features docs in the table of contents

#### Tests
Clicked the links.

#### Documentation
Is documentation.

#### Primary reviewers
@ernopp (@anne-edwards )

### CODE REVIEW

File: docs/content/faqs/key-features.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+#### GameObjects are optional
+
+You only use GameObjects when you want to. You have the option to do computation on entity data without the overhead of MonoBehaviours. Built-in functionality, such as the spawning pipeline, makes use of this.   
+
+## Customizability
+
+#### Full source access
+
+At Improbable, we’ve always believed that everyone benefits from open development and that code gets better, faster, when it’s public. We’re developing the SpatialOS GDK for Unity in the open so that you can better understand it, and directly impact its development. We welcome [issue reports](https://github.com/spatialos/UnityGDK/issues) and [feature requests](https://github.com/spatialos/UnityGDK#give-us-feedback) right now, and when we enter alpha, we’ll being accepting pull requests directly from the community.
```
Comment (jessicafalk) [inline docs/content/faqs/key-features.md]: `we’ll being accepting pull requests directly from the community.`that sentence sounds weird. `well start accepting ...`?

File: docs/content/faqs/faqs.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+It is a means of separating data (Entities and Components) from logic that works on this data (Systems). This allows you to allocate and process data more efficiently, as well as build logic systems to handle this data separately.
+
+With our GDK for Unity, you have two workflow options to build your gameplay features:
+
+1. A MonoBehaviour-centric workflow: takes advantage of Unity’s fully developed MonoBehaviour tooling, workflow and APIs. 
+
+2. An ECS workflow: takes advantage of the ECS development paradigm and associated performance improvements.
+
+Planned for our alpha release, our MonoBehaviour workflow supports an API similar to the [Readers and Writers API](https://github.com/spatialos/UnitySDK/blob/master/docs/interact-with-world/interact-components.md#example-of-monobehaviours-component-readers-and-writers) in the [SpatialOS SDK for Unity](https://github.com/spatialos/UnitySDK/blob/master/docs/interact-with-world/interact-components.md#example-of-monobehaviours-component-readers-and-writers). Follow [this Trello card](https://trello.com/c/ytJ6fZZT/10-monobehaviour-workflow) to keep track of our development progress.
```
[the remark this answers is not in the document]
Comment (ElleEss) [reply docs/content/faqs/faqs.md]: Good catch, thanks.

File: docs/content/faqs/faqs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+1. A MonoBehaviour-centric workflow: takes advantage of Unity’s fully developed MonoBehaviour tooling, workflow and APIs. 
+
+2. An ECS workflow: takes advantage of the ECS development paradigm and associated performance improvements.
+
+Planned for our alpha release, our MonoBehaviour workflow supports an API similar to the [Readers and Writers API](https://github.com/spatialos/UnitySDK/blob/master/docs/interact-with-world/interact-components.md#example-of-monobehaviours-component-readers-and-writers) in the [SpatialOS SDK for Unity](https://github.com/spatialos/UnitySDK/blob/master/docs/interact-with-world/interact-components.md#example-of-monobehaviours-component-readers-and-writers). Follow [this Trello card](https://trello.com/c/ytJ6fZZT/10-monobehaviour-workflow) to keep track of our development progress. 
+
+ 
+
+If you are interested in finding out more about our ECS workflow, these topics in our documentation are a good place to start: [Component updates](../component-data.md), [Events](../content/events.md), [Commands](../commands.md) and [Reactive components](../reactive-components.md).
```
[the remark this answers is not in the document]
Comment (ElleEss) [reply docs/content/faqs/faqs.md]: Thanks.

File: docs/content/faqs/key-features.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## Publicly-available experimental features
+
+#### Mobile game development
+
+To enable mobile game development support for SpatialOS, we have an [experimental branch](https://github.com/spatialos/UnityGDK/tree/experimental/android-support) for Android on the SpatialOS GDK. We plan to have iOS support for the GDK soon.
```
[the remark this answers is not in the document]
Comment (ElleEss) [reply docs/content/faqs/key-features.md]: Thanks

File: docs/content/faqs/key-features.md (hunk 1 above)
Comment (ElleEss) [reply docs/content/faqs/key-features.md]: Yes, it's a typo - good spot, eagle eyes.

**Pull request merged**